### DISCUSSION TITLE
CDM development environment

### DISCUSSION THREAD

**Discussion by melpignano** [finos/common-domain-model#3412] [Q&A]
hi all, i would like to create a development environment to perform unit test on the modify applied to the code. Is it possible to install a local or cloud environment? Is there documentation available on this topic?

**Comment by lolabeis** [comment]
Hi @melpignano Thank you for raising this question.

The CDM repository's README file provides some guidance about the development environment in the "Contributing" section. Rosetta provides a cloud-hosted solution. There is also an open-source VS code plugin for the DSL that allows to use VS code directly as a dev environment. This section will be augmented soon with further resources, as we're in the process of migrating over all of the CDM documentation to FINOS: see https://github.com/finos/common-domain-model/issues/1967.

Some of my colleagues will get in touch with you directly to understand your use-case and how we can assist in the mean-time.
